### DISCUSSION TITLE
Wiki Module

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3642] [General]
A 1.0 version of an open source Wiki module for Oqtane has been published here: 

https://github.com/oqtane/oqtane.wikis

It is still a work in progress but it contains the core functionality for managing wiki content. Whether a Wiki approach is appropriate for the full scope of Oqtane Framework documentation is yet to be determined. I mentioned at the last Developer Meeting that I would create a Wiki module so I wanted to follow through on that commitment... and I do intend to use it to provide documentation on the Oqtane Marketplace site.

**Comment by thabaum** [comment]
I feel it can be a value to the Oqtane community learning all kinds of stuff.  Can you open up discussions on each main repository such as Blogs and Wiki?  This way we can discuss the modules themselves without putting up issues.

I look forward to seeing the wiki get used to help produce content needed to administrate and develop using the Oqtane Framework.  I will be sure to test and assist where I can.

I am looking forward to the next meeting, as I would love to see a Learn.Oqtane.org site created to link all these resources similar to Microsoft Learn but in our own community version.   We can take ideas from here as we can have a Wiki Page that is dedicated to Shows and have a link going to this resource displaying links and info to shows that spotlight Oqtane or .NET even.  A simple landing page that showcases all the different resources for learning Oqtane.  

I also suggested a "Learn" module to create/use but really the HTML module can do it all for the Learn.Oqtane.org site, this site can be static and serve as a test site for more static type of content.

This way, whatever project you feel helps you the most and want to contribute to you have options.  I think they will all come in handy.  I was always confused with the Wiki articles vs Doc articles and Blog Articles, but I get the point of them all now.

We all have ways we like to contribute, so this should open up the doors for those that feel this is the way to go!

Thank you, cheers!

**Comment by sbwalker** [comment]
The 1.0 version of the Wiki module is now available on the Marketplace. You can install it directly from your Oqtane installation if you want to try it out. Features were added for tagging wiki pages and automatically maintaining the link graph between pages. Example screen shots were also added to the repo README: https://github.com/oqtane/oqtane.wikis
